tests/static-api/_expected/v1/teams/leaderless.json

```diff
@@ -13,7 +13,17 @@
   ],
   "alumni": [],
   "github": null,
-  "website_data": null,
+  "website_data": {
+    "name": "Leaderless",
+    "description": "Test",
+    "page": "leaderless",
+    "email": null,
+    "repo": null,
+    "discord": null,
+    "zulip_stream": null,
+    "matrix_room": null,
+    "weight": 0
+  },
```

Comment:
This file and two others are missing a trailing newline.  It was preexisting, but since we're touching it anyway, maybe it makes sense to fix this.

Reply:
These files are generated (using the same system for generating the static API), and I would not want to make that kind of unrelated change in this PR.